README.md

```diff
@@ -58,6 +63,16 @@ const db = createKeyValueTable(REDIS, options)
   - `connectionString` [redis:// connection string](http://www.iana.org/assignments/uri-schemes/prov/redis)
   - `ioredisOptions` Options to pass along to [ioredis](https://github.com/luin/ioredis)
 
+#### MEMORY
+
+```js
+const db = createKeyValueTable(MEMIRY, options)
+```
+
+- `options`
+  - `serializer` function used to serialize data before storing, may return a Promise, defaults to `JSON.stringify`
```

Comment:
why do we serialize? or is it persisted between refreshes?

Reply:
Makes sure the database's contents cannot be changed by maintaining a reference to the data and is more consistent with other store backends.